src/openapi/streaming/connection/connection.js

```diff
@@ -60,7 +74,11 @@ function onTransportFail(error) {
     this.transport.setConnectionSlowCallback(this.connectionSlowCallback);
 
     if (this.state === STATE_STARTED) {
-        this.transport.updateQuery(this.authToken, this.contextId);
+        this.transport.updateQuery(

```

Comment:
no changes are needed in legacy signalr? or did I miss it?

Reply:
legacy signalr only use first 2 params